src/pages/settings/Subscription/CardSection/utils.ts

```diff
@@ -116,6 +120,7 @@ function getBillingStatus(
                 title: translate('subscription.billingBanner.retryBillingError.title'),
                 subtitle: translate('subscription.billingBanner.retryBillingError.subtitle'),
                 isError: true,
+                isRetryAvailable: false,
```

Comment:
We sure we can’t retry the billing if the card is expiring soon? 

I know we have error type for this but I wasn’t sure if retry will be blocked due to this.

Reply:
We're not returning error when card is expiring soon so there wont be anything to retry.

Im not sure if this comment is in the right place as this line is returned in case of SubscriptionUtils.PAYMENT_STATUS.RETRY_BILLING_ERROR

Reply:
Yeah the line is incorrect. Let me put it at the right place.